module/plugins/PluginManager.py

```diff
@@ -239,15 +246,15 @@ def parseUrls(self, urls):
         res = [] # tupels of (url, plugin)
 
         for url in urls:
-            if type(url) not in (str, unicode, buffer): continue
+            if type(url) not in (str, memoryview): continue
```

Comment:
Codacy found an issue: [Using type() instead of isinstance() for a typecheck.](https://app.codacy.com/manual/pyLoad/pyload/pullRequest?prid=4846282)